src/section-factoring-special-polynomials.ptx

```diff
@@ -311,16 +303,16 @@
                 </ul></p>
             </statement>
         </definition>
-
+<!-- todo: this aside rendered poorly. It was opaque and behind the main text. I then expanded the window and compressed it and it rendered properly. JUst double check that it is rendering properly in the future. -->
         <aside>
-            <title>Sums of Cubes <em>are</em> Differences of Cubes</title>
-            <p>Technically a sum of cubes, <m>a^3+b^3</m>, is equal to <m>a^3-(-b)^3</m>. If that is confusing, then forget about this and continue. If you understand this, then you can treat any sum of cubes as a difference of cubes where <m>b</m> is negative, and you only need to memorize the <em>difference</em> of cubes formula.</p>
+            <title>Differences of cubes <em>are</em> sums of cubes</title>
```

Comment:
It looks like you have a preference for the sum of cubes over the difference of cubes. Possibly because `+` is the "primary" operation, and `-` is "secondary". It's no big deal, but there are reasons to go the other way. The difference of cubes more closely parallels the difference of squares. Also, with the difference of cubes formula, within each factor all the signs are the same, so it strikes me as an easier formula to memorize. Thoughts?